tests/framework/utils.py

```diff
@@ -190,3 +191,41 @@ def run_cmd(cmd, ignore_return_code=False, no_shell=False):
         run_cmd_async(cmd=cmd,
                       ignore_return_code=ignore_return_code,
                       no_shell=no_shell))
+
+
+def get_threads(pid):
+    """
+    Return dict consisting of child threads.
+
+    {
+        "name": thread_pid
+    }
+    """
+    
+    ret_val = {}
```

Comment:
I would call this `threads_map` or something more specific than `ret_val`.

Reply:
Done.